### PR TITLE
5.0 Beta 1.1 Release Notes

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.1.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+#### State Relevancy Flag
+
+Transactions were finalized differently by the `UtxoReceivedFinalityFlow` and `UtxoFinalityFlow` causing the relevancy flag to be inconsistently set. 
+As a result, for the initiating participant on a transaction, a Relevant State could incorrectly be flagged as available.
+As of this release, both `UtxoReceivedFinalityFlow` on the counterparty side and `UtxoFinalityFlow` on the initiating side correctly persist State relevancy when the transaction is verified. 
+
+#### ContractState Cast Exception
+
+An exception was incorrectly thrown if an attempt was made to cast a `ContractState`.
```
Comment (vlajos) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.1.md:54]: An exception was incorrectly thrown if an attempt was made to call `getInputStateAndRefs(…)`.

Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.1.md]: Thanks!

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.1.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### CSDE 
+
+#### Beta 1.1 API
+
+The [CorDapp Standard Development Environment (CSDE)](cordapp-standard-development-environment/csde.html) has been upgraded to use the Corda 5.0 Beta 1.1 API. 
+
+#### Example UTXO Ledger CorDapp
+
+The CSDE now includes an [example  UTXO (unspent transaction output) ledger CorDapp](....). This simple chat application  is available in both the [CSDE-cordapp-template-kotlin](https://github.com/corda/CSDE-cordapp-template-kotlin) and [CSDE-cordapp-template-java](https://github.com/corda/CSDE-cordapp-template-java) repositories.
```
Comment (yarivus) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.1.md:30]: redundant spaces:
```
application  is available
```

**Review by yarivus — CHANGES_REQUESTED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — COMMENTED** [review]
LGTM

**Review by blsemo — APPROVED** [review]
The ledger bugfix section looks good to me.

**Review by colin-hillis — APPROVED** [review]
LGTM

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**